mlrun/__main__.py

```diff
@@ -1058,30 +1063,32 @@ def project(
                 },
             )
         try:
-            proj.run(
-                run,
-                workflow_path,
-                arguments=args,
-                artifact_path=artifact_path,
-                namespace=namespace,
-                sync=sync,
-                watch=watch,
-                dirty=dirty,
-                workflow_handler=handler,
-                engine=engine,
-                local=local,
-                schedule=schedule,
-                timeout=timeout,
-            )
+            try:
+                proj.run(
+                    run,
+                    workflow_path,
+                    arguments=args,
+                    artifact_path=artifact_path,
+                    namespace=namespace,
+                    sync=sync,
+                    watch=watch,
+                    dirty=dirty,
+                    workflow_handler=handler,
+                    engine=engine,
+                    local=local,
+                    schedule=schedule,
+                    timeout=timeout,
+                    overwrite=overwrite_schedule,
+                )
+            except mlrun.errors.MLRunConflictError as error:
+                # In this way we are updating only the conflict error that is relevant to schedules
+                raise mlrun.errors.MLRunConflictError(
+                    str(error).replace("overwrite = True", "--overwrite-schedule")
+                )
```

Comment:
maybe you can do something like `error.msg.replace()` I am not sure what is the output you are getting by doing what you implemented but seems like you'll have 2 `MLRunConflictError` in the error message. If that is not the case, ignore.